src/gpo/assets/en-US/PowerToys.adml

```diff
@@ -69,13 +70,19 @@ If enabled, the notification is suspended.
 
 If disabled or not configured, the notification is shown.
 
-Note: The notification about new major versions is always displayed.
+Notes: The notification about new major versions is always displayed. This policy has no effect if the update notifications are disable.
```

Comment:
Q: does "disabled" mean disabled by user, or by policy?

Reply:
Both. If the user or the other policy didables the toast notification then you can't suspend it because it is already disabled.

I know this is logical but it is the normal thing to mention other policies/settings that make a policy not working.